scipy/sparse/linalg/_isolve/tests/test_minres.py

```diff
@@ -6,83 +6,94 @@
 from pytest import raises as assert_raises
 
 
-def get_sample_problem():
+def get_sample_problem(make_complex=False):
```

Comment:
minor: I suppose this is getting called 8 times now, to produce only two unique values, could decorate with a cache perhaps.